src/test/java/core/basesyntax/RegistratonTest.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class RegistratonTest {
+    private static RegistrationServiceImpl registrationService;
+    private static StorageDao storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.PEOPLE.add(new User("user123", "password123", 25));
+        Storage.PEOPLE.add(new User("user456", "password456", 30));
+    }
+
+    @Test
+    void register_LoginNotNull_notOk() {
+        String login = "user123";
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User(login, "password", 20)));
+        assertNotNull(storageDao.get(login));
```

Comment:
**LoginNotNull**_notOk?
LoginNotNull is not a valid state? 
 register_**LoginAlreadyExist**_notOk maybe?